HarmonyTests/Tools/TestAccessTools.cs

```diff
@@ -288,5 +289,97 @@ public void Test_AccessTools_FieldRefAccess_Anonymous()
 			value = newValue;
 			Assert.AreEqual(newValue, fieldInfo.GetValue(instance));
 		}
+
+		[Test]
+		public void Test_AccessTools_MethodDelegate()
```

Comment:
I like that this does not end up like 100 tests - faster and in this case just fine